### ISSUE TITLE
Allow Broadcasting Emails to 1 Person To Avoid Footer

### ISSUE THREAD

**Issue by ericlanigan** [Simplero/Roadmap#468]
Basically, it's a pain in the ass to schedule emails to send to people in general... so I started using Simplero to send reminder emails to people months from now. That said, I'm pretty sure they'll show up with the footer rather than just a plain text email. It'd be nice for one contact broadcasts to not have that show up... to essentially allow Simplero to act have the "send future emails" functionality a lot of external and specific apps charge for. 

This should probably be really low priority, but I thought it would be nice. 

All of this is moot if there's a way I can turn off the footer and make my emails send like normal emails... but I don't think I can. 

Thanks!
Eric

**Comment by calvincorreli** [comment]
Hm. So what you're saying is you want to schedule a simple email to a specific contact on a specific time?

What are these emails for? Just an example, so I get a sense of what you're looking to accomplish here.

What are some of the external and specific apps you could pay for that would do this for you? That, too, could help me better understand.

**Comment by ericlanigan** [comment]
Hi Calvin,

I could have sworn this was something I looked into a few months ago with apps that were charging people ~$10-$15/mo. And now I google it to find those apps and I find a free app (Spark email) that does exactly what I'm talking about and it's free. 

I closed the issue. Thank you for being attentive and inquiring further. 

Best,
Eric


Eric Lanigan
314.324.0422
EricLanigan.com
ClearlyMotivated.com
